jenkins/opensearch/distribution-build.jenkinsfile

```diff
@@ -52,6 +52,41 @@ pipeline {
                             } else {
                                 echo "Skipping publishing snapshots, builds/opensearch/maven does not exist."
                             }
+
+
+                            if (fileExists("$WORKSPACE/builds/opensearch/dist")){
```

Comment:
Why add this check? I think we should always expect these artifacts to be created.  If this didn't happen shouldn't the build fail?

Reply:
It was just a check. will remove it.

Reply:
Removed